src/connectedk8s/setup.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from codecs import open
+from setuptools import setup, find_packages
+try:
+    from azure_bdist_wheel import cmdclass
+except ImportError:
+    from distutils import log as logger
+    logger.warn("Wheel is not available, disabling bdist_wheel hook")
+
+# TODO: Confirm this is the right version number you want and it matches your
+# HISTORY.rst entry.
+VERSION = '0.1.4'
+
+# The full list of classifiers is available at
+# https://pypi.python.org/pypi?%3Aaction=list_classifiers
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'License :: OSI Approved :: MIT License',
+]
+
+# TODO: Add any additional SDK dependencies here
+DEPENDENCIES = [
+    'azure-cli-core',
+    'kubernetes',
+    'knack==0.7.0rc4',
+    'pycryptodome',
+    'websocket-client~=0.56.0'
```

Comment:
You don't need to include dependencies that is already introduced in [azure-cli](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/setup.py#L52) or [azure-cli-core](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/setup.py#L52). You can remove azure-cli-core, knack and websocket-client.